progress.go

```diff
@@ -0,0 +1,47 @@
+package ghostferry
+
+import (
+	"github.com/siddontang/go-mysql/mysql"
+)
+
+const (
+	TableActionWaiting   = "waiting"
+	TableActionCopying   = "copying"
+	TableActionCompleted = "completed"
+)
+
+type TableProgress struct {
+	LastSuccessfulPK uint64
+	TargetPK         uint64
+	CurrentAction    string // Possible values are defined via the constants TableAction*
+}
+
+type Progress struct {
+	// Possible values are defined via the constants State*
```

Comment:
Maybe mention these are in ferry.go (as much as that may age), because that file isn't modified, so it won't show up in the diff.